katas/langtonant/charles/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Hi Charles, please update your code so it doesn't need to use an html page, it should be enough to print in console the final state of the grid after ant takes the given number of steps

Reply:
Ok, I'll do it. Thanks